pkg/exchange/bybit/stream.go

```diff
@@ -0,0 +1,94 @@
+package bybit
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/exchange/bybit/bybitapi"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+const (
+	// Bybit: To avoid network or program issues, we recommend that you send the ping heartbeat packet every 20 seconds
+	// to maintain the WebSocket connection.
+	pingInterval = 20 * time.Second
+)
+
+type Stream struct {
+	types.StandardStream
+}
+
+func NewStream() *Stream {
+	stream := &Stream{
+		StandardStream: types.NewStandardStream(),
+	}
+
+	stream.SetEndpointCreator(stream.createEndpoint)
+	stream.SetParser(stream.parseWebSocketEvent)
+	stream.SetDispatcher(stream.dispatchEvent)
+
+	stream.OnConnect(func() {
```

Comment:
OnConnect will be triggered everytime it's connected or re-connected

Reply:
fixed by https://github.com/c9s/bbgo/pull/1265/commits/f9b9218e9dd5815cd6cae453b2428f7e54c5d5e6